widgets/common/src/deployments/DeploymentActionsMenu.tsx

```diff
@@ -11,24 +13,40 @@ export const actions = Object.freeze({
     update: 'update'
 });
 
+const executeWorkflowPermission = 'execution_start';
 const translate = Stage.Utils.getT('widgets.common.deployments.actionsMenu');
-const menuItems = [
-    { name: actions.install, icon: 'play', permission: 'execution_start' },
+type MenuItem = { name: string; icon: string; permission: string };
+const menuItems: MenuItem[] = [
+    { name: actions.install, icon: 'play', permission: executeWorkflowPermission },
     { name: actions.update, icon: 'edit', permission: 'deployment_update_create' },
     { name: actions.setSite, icon: 'building', permission: 'deployment_set_site' },
     { name: actions.manageLabels, icon: 'tags', permission: 'deployment_create' },
-    { name: actions.uninstall, icon: 'recycle', permission: 'execution_start' },
+    { name: actions.uninstall, icon: 'recycle', permission: executeWorkflowPermission },
     { name: actions.delete, icon: 'trash alternate', permission: 'deployment_delete' },
     { name: actions.forceDelete, icon: 'trash', permission: 'deployment_delete' }
 ];
 
+function isAvailable(item: MenuItem, workflows: Workflow[]) {
+    if (item.permission === executeWorkflowPermission) {
+        const workflow = find(workflows, { name: item.name });
```

Comment:
What do you think about using native `Array.prototype.find` function?
(as suggested by [UI Code Style Guidelines](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/435388492/UI+Code+Style#SHOULD%3A-Use-built-in-JS-functions-instead-of-utilities-functions-from-external-packages))

```suggestion
        const workflow = workflows.find(workflow => workflow.name === item.name);
```

Reply:
Make sense, applied.